src/_help/customization-options/logo-meta-images.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Logo and Meta images
+---
+
+- TOC
+{:toc}
+
+## Logo and favicon
+
+You can upload your own logo and favicon on Bump.sh to customize your documentation with your brand or project colors.
+
+From the "Customize UI" tab in the documentation settings, you can add your own logo and favicon.
+
+![](/images/help/logo-favicon.png)
+
+To provide the best image quality, it is recommended to adhere to the dimensions below:
+- Logo:
+- Favicon:
```

Comment:
You might not want to present the favicon in this list because we don't recommend a favicon size, as it may vary depending on the browser.
Ideally, one should provide different sized-favicons to fit every type of screens, but we don't support that.